lib/interface_quda.cpp

```diff
@@ -899,7 +899,14 @@ void loadCloverQuda(void *h_clover, void *h_clovinv, QudaInvertParam *inv_param)
 
   CloverFieldParam clover_param;
   clover_param.nDim = 4;
-  clover_param.csw = inv_param->clover_coeff;
```

Comment:
@cpviolator In response to https://github.com/lattice/quda/pull/1160#issuecomment-870927989 I get that external codes can now specify either csw and kappa or `clover_coeff` directly (as we do in tmLQCD). I'm wondering, however, about the change of meaning of `clover_param.csw` here. Clearly, this is an improvement as `csw` should always just be `csw` for consistency.

I'm just not sure if anyone possibly relies on `clover_param.csw` being `csw*kappa`.